src/api/controllers/UserController.ts

```diff
@@ -60,13 +60,20 @@ export class UserController {
   @Get('save/userId/:userId/postId/:postId/') 
   async savePost(@Params() params: PostAndUserUuidParam): Promise<GetPostResponse | ErrorResponse> {
     try {
```

Comment:
this try catch function should be put into a utils since it looks like this exact structure is used everywhere in the code except that the key changes every now and then (post: ... or user:...).